host_core/test/fixtures/actors/kvcounter_unpriv/README.md

```diff
@@ -0,0 +1,23 @@
+# KVCounter Unpriviledged
```

Comment:
It feels weird to me that we have an entirely different actor folder for the unprivileged actor. Is there something preventing us from just doing `make unpriv` on the original kvcounter actor makefile?

Reply:
We could do `make unpriv` if we built that in, but I feel like that gets into us making unnecessary modifications to our examples when it's only used for tests. The motivation for this is really just so that we can run `make` here and get our unpriv actor automatically built and put in place for testing.

The alternative to this would just be to go to the `examples` repo whenever we need to update this actor, comment out the `keyvalue` capability, build it, move it to fixtures, and then replace the public key in the `Constants` module. I'm fine with either way

Reply:
For now, I think what you've got here works best since it's easily automatable with all the other samples.